examples/discord_links.py

```diff
@@ -12,36 +14,40 @@ async def main():
 
     # add a link
     await client.add_link(player_tag, discord_id)
-    print("Player Tag {} is now linked to discord id {}".format(player_tag, discord_id))
+    print(f"Player Tag {player_tag} is now linked to discord id {discord_id}")
 
     # get a link by tag
     discord_id = await client.get_link(player_tag)
-    print("Player Tag {} is linked to discord id {}".format(player_tag, discord_id))
+    print(f"Player Tag {player_tag} is linked to discord id {discord_id}")
 
     # update a link
     new_discord_id = 230214242618441728
     await client.delete_link(player_tag)
     await client.add_link(player_tag, new_discord_id)
-    print("Link for player tag {} has been updated to have discord id {}".format(player_tag, new_discord_id))
+    print(f"Link for player tag {player_tag} has been updated to have discord id {new_discord_id}")
 
     # delete a link
     await client.delete_link(player_tag)
-    print("Link for player tag {} has been removed from the database.".format(player_tag))
+    print(f"Link for player tag {player_tag} has been removed from the database.")
 
     # batch get links by tag
     player_tags = ["#JY9J2Y99", "#2GV0QY8G8", "#PP9L22C8", "#2LPC9J8L"]
     links = await client.get_links(*player_tags)
     for tag, discord_id in links:
         if discord_id is None:
-            print("Player tag {} doesn't have any links.".format(tag))
+            print(f"Player tag {tag} doesn't have any links.")
         else:
-            print("Player tag {} is linked to discord id {}".format(tag, discord_id))
+            print(f"Player tag {tag} is linked to discord id {discord_id}")
 
     # batch get links by id
-    discord_ids = [246286410946969610, 230214242618441728, 267057699856842753]
+    discord_ids = [
```

Comment:
Why did you split this into 3 separate lines? I think it's more than readable as just 1, and doesn't go anywhere near 120chars.